docs/other_resources/high_level_architecture.md

```diff
@@ -35,14 +35,19 @@ process gets pushed to event subscribers.
 
 ## Readiness & Liveness probes
 
+### HTTP
+
 Flagd exposes HTTP liveness and readiness probes.
 These probes can be used for K8s deployments.
-With default
-start-up configurations, these probes are exposed at the following URLs,
+With default start-up configurations, these probes are exposed on the metrics port (default: 8014) at the following URLs,
 
 - Liveness: <http://localhost:8014/healthz>
 - Readiness: <http://localhost:8014/readyz>
 
+### gRPC
+
+Flagd exposes a [standard gRPC liveness check](https://github.com/grpc/grpc/blob/master/doc/health-checking.md) on the metrics port (default: 8014).
```

Comment:
Can you include a sample path like we have for the HTTP probe?

Reply:
I also agree with @toddbaert here. 

We cannot specify gRPC URL without the schema. IMO having the link to health-check documentation [^1] should be sufficient. But may be we can add the line (extracted from reference doc)

> "A client can query the server’s health status by calling the Check method, and a deadline should be set on the rpc"

And say we do not use/support `service` filtering  

[^1]: https://github.com/grpc/grpc/blob/master/doc/health-checking.md

Reply:
I am resolving this for now @beeme1mr . I'll open a new PR if you think additional doc is needed.